.github/workflows/post-jobs-slack.yaml

```
@@ -24,6 +24,10 @@ jobs:
           deploy: false
           test: true
 
+          # Also deploy to Mastodon (all secrets required in repository secrets)
+          mastodon_deploy: true
```

```suggestion
          mastodon_deploy: false
```
We could have this true if you want to provide a token you are using, but understandable if not! It would run each time we do the test.

---

I'd rather not put a token here simply because I don't want it to become a permanent dependency for this workflow.